src/cursor/cur_backup_incr.c

```diff
@@ -171,6 +177,8 @@ __curbackup_incr_next(WT_CURSOR *cursor)
                 if (F_ISSET(cb, WT_CURBACKUP_RENAME) ||
                   (F_ISSET(cb, WT_CURBACKUP_CKPT_FAKE) && F_ISSET(cb, WT_CURBACKUP_HAS_CB_INFO))) {
                     WT_ERR(__wt_fs_size(session, cb->incr_file, &size));
+                    __wt_verbose(session, 
+                        WT_VERB_BACKUP, "Set key with full copy size %" PRIuMAX, (uintmax_t) size);
```

Comment:
For this one and the one below I think it would be helpful to:
1. add `cb->incr_file` name into the message to know what file it is working on.
2. include `WT_BACKUP_FILE` in there for quick visual information without having to parse the details of the message.

Reply:
done 👍